application/views/css/layout.css

```diff
@@ -1185,6 +1185,41 @@ h1.emote b {
   color: #fff;
 }
 
+<<<<<<< HEAD
+=======
```

Comment:
Seems to have been a merge conflict. These 2 lines and the one at the bottom should be removed. Row 1188, 1189 and 1222